### ISSUE TITLE
initializer broken by ruby-jwt api changes 

### ISSUE THREAD

**Issue by subuta** [nsarno/knock#43]
I found [config.token_secret_signature_key](https://github.com/nsarno/knock/blob/11162c1549eabfdf1552b4d1c7dde993161ef9ff/lib/generators/templates/knock.rb#L77) was broken by ruby-jwt API updates ... 
```
# config.token_secret_signature_key = -> { JWT.base64url_decode Rails.application.secrets.auth0_client_secret }
```

Latest ruby-jwt(v1.5.3) completely removes `base64url_decode ` method from JWT module(at this [PR](https://github.com/jwt/ruby-jwt/commit/7a7ac9a339f0a2967c6d9bcea7f6ccaa3661cf7e))

BTW, We may use these work around as a temporary solution :)
(we may have better solution for this problem...)

```
require 'base64'

# extracted from original [method](http://www.rubydoc.info/github/jwt/ruby-jwt/JWT.base64url_decode)
config.token_secret_signature_key = -> {
    secret = Rails.application.secrets.auth0_client_secret
    secret += '=' * (4 - secret.length.modulo(4))
    Base64.decode64(secret.tr('-_', '+/'))
  }
```

**Comment by solsberg** [comment]
Thanks for the workaround, works for me also

**Comment by nsarno** [comment]
There's an issue open in `ruby-jwt` (https://github.com/jwt/ruby-jwt/issues/133) because of `1.5.3` breaking compatibility with `1.5.2`. The breaking version is going to be yanked and re-released as `1.6.0` so it should be all good after that.

I'll let this issue open for people to see the workaround in the meantime. Thanks for your help @subuta !

**Comment by nsarno** [comment]
https://github.com/jwt/ruby-jwt/issues/133
